Messages barrier max time (#141)
kibertoad/message-queue-toolkit: README.md

        * `messageTypeField` - which field in the message describes the type of a message. This field needs to be defined as `z.literal` in the schema and is used for resolving the correct schema for validation
        * `locatorConfig` - configuration for resolving existing queue and/or topic. Should not be specified together with the `creationConfig`.
        * `creationConfig` - configuration for queue and/or topic to create, if one does not exist. Should not be specified together with the `locatorConfig`.

>         * `messageTimestampField` - which field in the message contain the message creation date. This field needs to be ISO-8601 date string and by default it is `timestamp`, if your messages doesn't contain it the library will add one automatically to avoid infinite loops on consumer;

Reviewer: can we support Date too? nothing prevents zod schema from transforming timestamps within the schema

Author: Already done 🙇

>         * `messageTimestampField` - which field in the message contain the message creation date (by default it is `timestamp`). This field needs to be a `Date` object or ISO-8601 date string, if your messages doesn't contain it the library will add one automatically to avoid infinite loops on consumer;

Reviewer 2: 🟢 
```suggestion
        * `messageTimestampField` - which field in the message contains the message creation date (by default it is `timestamp`). This field needs to be a `Date` object or ISO-8601 date string, if your message doesn't contain it the library will add one automatically to avoid infinite loops on consumer;
```

    * `options`, composed by
        * `handlers` – configuration for handling each of the supported message types. See "Multi-schema handler definition" for more details;
        * `messageTypeField` - which field in the message describes the type of a message. This field needs to be defined as `z.literal` in the schema and is used for routing the message to the correct handler;

>         * `messageTimestampField` - which field in the message contain the message creation date (by default it is `timestamp`). This field needs to a `Date` object or ISO-8601 date string;

Reviewer 2: 🟢 
```suggestion
        * `messageTimestampField` - which field in the message contains the message creation date (by default it is `timestamp`). This field needs to be a `Date` object or an ISO-8601 date string;
```

Author: Thank you so much Daria!! 🧡

>         * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (avoid infinite loops). Default is 4 days;

Reviewer: we mark as consumed and not DLQ it?

Author: Yes, it goes to DLQ (only on SNS and AMQP) as any other message that can't be consumed, but only if DLQ is configured, already mentioned as a note on DLQ section 🙌 good catch Igor 🙏

>         * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (avoid infinite loops). Default is 4 days;

Reviewer: ```suggestion
        * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (and sending to DLQ, if one is configured). This is used to avoid infinite loops. Default is 4 days;
```
